### ISSUE TITLE
Harmonising the way "Sort id" works

### ISSUE THREAD

**Issue by giovanniborella** [mapcentia/geocloud2#49]
Using 'Sort id' on layers yields  this result (left part is Vidi client, right part is GC2 admin)
![billede](https://user-images.githubusercontent.com/18160930/46787878-0d9ca600-cd39-11e8-921b-b4de18cb53f4.png)

However; when using 'Sort id' on attributes to be used on click/Info:
![billede](https://user-images.githubusercontent.com/18160930/46787940-3f157180-cd39-11e8-80e4-f743a428795c.png)

TL;DR: **It seems that one of the sort orders needs to be reversed - to unify the UX.**

**Comment by mapcentia** [comment]
Yes, I know. We started out with a unified way to do it, but some users wanted the bigger sort ids on top in the layer tree. So I made a setting in app/conf/App.php where you can reverse the layer tree sorting (which is now the typical way to do it). To fix this we need to implement a setting for the attribute sorting also.
